ktor-io/common/src/io/ktor/utils/io/core/Input.kt

```diff
@@ -12,7 +12,6 @@ public abstract class Input(
     remaining: Long = head.remainingAll(),
     public val pool: ObjectPool<ChunkBuffer> = ChunkBuffer.Pool
 ) : Closeable {
-    private val state = InputSharedState(head, remaining)
```

Comment:
BTW, there are also:
* OutputSharedState
* BufferSharedState
* ByteChannelSequentialBaseSharedState

Reply:
BufferSharedState is still there :)